README.md

```diff
@@ -91,8 +91,6 @@ Example command:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
 👤 **Author1**
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you put your name here instead of Author1 to make your readme more descriptive and professional 👍